tests_v2/cmux.py

```
@@ -24,63 +24,185 @@
+        or name == "com.cmuxterm.app.nightly.sock"
+        or name.startswith("com.cmuxterm.app.nightly.")
+        or name == "com.cmuxterm.app.staging.sock"
+        or name.startswith("com.cmuxterm.app.staging.")
+    )
+
+
 def _read_last_socket_path() -> Optional[str]:
+    is_stable_client = not os.environ.get("CMUX_TAG")
     for marker_path in _LAST_SOCKET_PATH_FILES:
         try:
             with open(marker_path, "r", encoding="utf-8") as f:
```

Ruff (0.15.12)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)